docs/UserGuide.md

```diff
@@ -251,6 +262,74 @@ Format: `edit -friend INDEX n/NAME m/MOBILE_NUMBER p/PASSPORT_NUMBER​`
 
 Example: `edit -friend 3 n/John m/91234567 p/E7654321`
 
+##Show
+
+### 1. Showing Activity Tab
+
+Switches the current Ui view to show Activity tab under the travel plan object panel.
+This command has no effect in a wishlist as a wishlist only contains activities.
+
+Format/ Example: `show -activity`
+
+### 2. Showing Accommodation Tab
+
+Switches the current Ui view to show Accommodation tab under the travel plan object panel.
+This command has no effect in a wishlist as a wishlist does not contain Accommodation.
```

Comment:
Do we allow them to type this command when in wishlist? Or do we output error message?

Reply:
Error msg thrown